pkg/controller/machineset/controller.go

```diff
@@ -50,7 +50,7 @@ var stateConfirmationInterval = 100 * time.Millisecond
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager) error {
 	r := newReconciler(mgr)
-	return add(mgr, r, r.MachineSetToMachines)
+	return add(mgr, r, r.MachineToMachineSets)
```

Comment:
analogous to https://github.com/kubernetes-sigs/cluster-api/blob/master/pkg/controller/machinedeployment/controller.go#L258